IconDiffBot/Core/PayloadProcessor.cs

```diff
@@ -146,6 +135,28 @@ async Task RunCheck()
 					checkRunId = await gitHubManager.CreateCheckRun(repositoryId, installationId, ncr, cancellationToken).ConfigureAwait(false);
 				}
 
+				if (pullRequest.Title.Contains("[IDB IGNORE]", StringComparison.InvariantCultureIgnoreCase)) {
+					logger.LogWarning("Pull request has [IDB IGNORE] in title. Aborting.");
+					await gitHubManager.UpdateCheckRun(repositoryId, installationId, checkRunId.Value, new CheckRunUpdate {

```

Comment:
Just extract the await to a separate method so you're not copypasting this.

Reply:
Considering this would need to take 5 different params just for a neutral result, and more for anything else, this seems to be fine.